### PR TITLE
Update Water Heater=Other Fuel to map to no water heater

### PR DESCRIPTION
**Pull request by joseph-robertson**
## Pull Request Description

Options here are to either:
- [ ] ignore this for now and delete this pull request (worry about it in ResStock-HPXML)
- [ ] explore setting fuel type to OtherFuel1
- [ ] modify the water heater tsv files to move Other Fuel probability over to the remaining columns
- [x] update cw, cd, dw, fixtures, distribution measures to skip when not finding a water heater
- [ ] other?

The important thing to note here is that when upgrading the water heater, the apply logic must include `!Water Heater|Other Fuel` so that we're not ADDING a previously nonexistent water heater.

## Checklist

Not all may apply:

- [ ] Unit tests have been added or updated
- [ ] All rake tasks have been run, and pass
- [ ] PAT project measures/outputs have been updated
- [ ] Documentation has been modified appropriately
- [ ] Any new options are added to `project_testing`
- [ ] `project_testing` runs without any failures
- [ ] No unexpected regression test changes
- [ ] All tests are passing (green) on circleci
- [ ] The [changelog](https://github.com/NREL/OpenStudio-BuildStock/blob/master/CHANGELOG.md) has been updated appropriately
- [ ] This branch is up-to-date with master

For more information on how to perform these checklist items, see the documentation's [Advanced Tutorial](https://resstock.readthedocs.io/en/latest/advanced_tutorial/index.html).

### CODE REVIEW

File: project_multifamily_beta/housing_characteristics/Water Heater.tsv [hunk 1]
```diff
@@ -1,61 +1,61 @@
...
-Propane	CR07	0.330000	0.000000	0.000000	0.030000	0.000000	0.000000	0.000000	0.000000	0.000000	0.650000	0.000000	0.000000	0.000000
-Propane	CR08	0.620000	0.000000	0.000000	0.000000	0.000000	0.000000	0.020000	0.000000	0.000000	0.340000	0.000000	0.030000	0.000000
-Propane	CR09	0.670000	0.000000	0.000000	0.000000	0.000000	0.000000	0.000000	0.000000	0.000000	0.310000	0.000000	0.020000	0.000000
-Propane	CR10	0.300000	0.000000	0.000000	0.000000	0.000000	0.000000	0.000000	0.000000	0.000000	0.700000	0.000000	0.000000	0.000000
-Propane	CR11	0.250000	0.000000	0.000000	0.030000	0.000000	0.000000	0.000000	0.000000	0.000000	0.690000	0.000000	0.000000	0.030000
+Dependency=Heating Fuel	Dependency=Location Region	Option=Electric Standard	Option=Electric Premium	Option=Electric Tankless	Option=Gas Standard	Option=Gas Premium	Option=Gas Tankless	Option=Oil Standard	Option=Oil Premium	Option=FIXME Oil Indirect	Option=Propane Standard	Option=Propane Premium	Option=Propane Tankless	Option=Other Fuel
+Electricity	CR02	0.780000	0.000000	0.000000	0.150000	0.000000	0.000000	0.000000	0.000000	0.000000	0.060000	0.000000	0.010000	0.000000
```
Comment (shorowit) [inline project_multifamily_beta/housing_characteristics/Water Heater.tsv]: @joseph-robertson Why does every line show up as changed? New lines? I thought Tony's script was specifically to address this.

Comment (joseph-robertson) [reply project_multifamily_beta/housing_characteristics/Water Heater.tsv]: good question

Comment (joseph-robertson) [reply project_multifamily_beta/housing_characteristics/Water Heater.tsv]: looks ok for singlefamilydetached Water Heater...

**Comment by joseph-robertson** [comment]
> Looks like this has some clothes washer changes and other lighting schedule changes too. Intentional?

Good question. I'm guessing it shouldn't be there. I'll look into it.

**Comment by joseph-robertson** [comment]
> > Looks like this has some clothes washer changes and other lighting schedule changes too. Intentional?
> 
> Good question. I'm guessing it shouldn't be there. I'll look into it.

Oh. I think I just need to merge master in. Some files had inadvertently not been updated on master some weeks ago.

**Pull request merged**